### ISSUE TITLE
Crash when started from command line with 640x480 or higher 4:3 resolution

### ISSUE THREAD

**Issue by orbea** [GLupeN64/GLupeN64#109]
OS: `Slackware64-current`
`mesa-2016.11.10_9557147_master-x86_64-1_git`
`RetroArch-2016.11.19_9b69d69_master-x86_64-1_git`
`GLupeN64-2016.11.19_22ebeb1_master-x86_64-1_git`

When starting GLupeN64 from a terminal with: `retroarch -L glupen64_libretro.so /path/to/game` it will crash if the 4:3 resolution is `640x480` or higher. The default resolution of `320x180` does not crash, however strangely enough it does not crash if opened from the RetroArch menu instead.
```
Thread 1 "retroarch" received signal SIGSEGV, Segmentation fault.
0x00007ffff1e03ba0 in __memcpy_ssse3 () from /usr/lib64/debug/libc.so.6
(gdb) bt
#0  0x00007ffff1e03ba0 in __memcpy_ssse3 () from /usr/lib64/debug/libc.so.6
#1  0x00007fffe8c89840 in OGLRender::updateBO (
    this=0x7fffe9484a10 <OGLVideo::get()::video+1104>, buffer=4, size=1048576,
    count=1, pointer=0x1739ab0) at GLideN64/src/OpenGL.cpp:707
#2  0x00007fffe8d2551e in InitZlutTexture ()
    at GLideN64/src/OGL3X/GLSLCombiner_ogl3x.cpp:146
#3  0x00007fffe8d25a87 in InitShaderCombiner ()
    at GLideN64/src/OGL3X/GLSLCombiner_ogl3x.cpp:243
#4  0x00007fffe8c61922 in Combiner_Init () at GLideN64/src/Combiner.cpp:77
#5  0x00007fffe8c908db in OGLRender::_initData (
    this=0x7fffe9484a10 <OGLVideo::get()::video+1104>)
    at GLideN64/src/OpenGL.cpp:2402
#6  0x00007fffe8c873be in OGLVideo::start (
    this=0x7fffe94845c0 <OGLVideo::get()::video>) at GLideN64/src/OpenGL.cpp:156
#7  0x00007fffe8caefee in PluginAPI::RomOpen (
    this=0x7fffe9484058 <PluginAPI::get()::api>)
    at GLideN64/src/common/CommonAPIImpl_common.cpp:190
#8  0x00007fffe8caf364 in RomOpen () at custom/GLideN64/MupenPlusPluginAPI.cpp:9
#9  0x00007fffe8d4a050 in main_run () at custom/mupen64plus-core/main/main.c:316
#10 0x00007fffe8d4a7c9 in CoreDoCommand (Command=M64CMD_EXECUTE, ParamInt=0,
    ParamPtr=0x0) at custom/mupen64plus-core/api/frontend_libretro.c:170
#11 0x00007fffe8da483e in EmuThreadFunction () at libretro/libretro.c:241
#12 0x00007fffe8d293a3 in co_init () at libretro-common/libco/amd64.c:116
#13 0x0000000000000000 in ?? ()
```
GDB log - http://pastebin.com/2GxhpLV3

**Comment by orbea** [comment]
I checked to make sure and this of course does not happen with the llvmpipe...

apitrace - http://ks392457.kimsufi.com/orbea/stuff/trace/libretro/glupen64-640x480-crash.trace.xz
log - http://pastebin.com/Ec6neMZK

**Comment by orbea** [comment]
First bad commit.
```
$ git bisect good
298c49fcb5d94879e41fc9af619ea9113a5351f9 is the first bad commit
commit 298c49fcb5d94879e41fc9af619ea9113a5351f9
Author: Logan McNaughton <logan@bacoosta.com>
Date:   Thu Nov 17 14:50:07 2016 -0800

    Use PBO for glTexSubImage2D

:040000 040000 42591a35a0d0c383a4295e0aee181d47b851faf0 f86bd13a35a6733c82bc7d87582f8633faf355f4 M	GLideN64
:040000 040000 219ecc9de306cc562d4030fd62b2697df7860c25 83abab5cc51850ef12da9262cbd197c6dc40fbdb M	custom
```
298c49fcb5d94879e41fc9af619ea9113a5351f9

**Comment by loganmc10** [comment]
I'm hoping this is resolved by: a13fcdf67961c4f621ae0881dae90bff172022b6, it fixes Works Armageddon for me, unfortunately I don't have a Windows system or a system with OGL 4+ to test on the weekends

**Comment by orbea** [comment]
Yes, this is fixed here. Thanks!
